app/controllers/offers_controller.rb

```diff
@@ -21,9 +27,27 @@ def create
     end
   end
 
+  def update
+    if @offer.update(offer_params)
+      redirect_to @offer, notice: 'Offer was successfully updated.'
```

Comment:
I'm not sure if notices are the best way to show this information. I think it's a design decision that should be discussed.

Reply:
What are other options?